bin.js

```diff
@@ -18,7 +18,7 @@ const shop = adventure({
 
 shop.add('Introduction: UCAN do it!', () => IntroductionProblem)
 shop.add('Lets space out', () => SpaceOutProblem)
-shop.add('Storing cat gifs for fun and gossip', () => CatGifsProblem)
+shop.add('Storing cat memes for fun and gossip', () => CatMemesProblem)
```

Comment:
You need to rename `CatGifsProblem` in the imports above also if you rename it here!